public-ui/src/app/data-browser/views/survey-view/components/survey-question-react.component.tsx

```diff
@@ -0,0 +1,137 @@
+import { HighlightReactComponent } from 'app/shared/components/highlight-search/HighlightReactComponent';
+import { reactStyles } from 'app/utils';
+import { ClrIcon } from 'app/utils/clr-icon';
+import { environment } from 'environments/environment';
+import { Configuration, DataBrowserApi } from 'publicGenerated/fetch';
+import * as React from 'react';
+import { SurveyAnswerReactComponent } from './survey-answer-react.component';
+
+
+const api = new DataBrowserApi(new Configuration({ basePath: environment.publicApiUrl }));
+
+const styles = reactStyles({
+    seeAnswers: {
+        fontSize: '14px',
+        cursor: 'pointer',
+        color: '#216fb4',
+    },
+    clIcon: {
+        width: '1.3em',
+        height: '1.3em',
+        color: '#216fb4'
+    }
+});
+
+
+interface Props {
+    isCopeSurvey: boolean;
+    question: any;
+    participantCount: number;
+    surveyConceptId: number;
+    searchTerm: string;
+}
+
+interface State {
+    showAnswers: boolean;
+    questionWithResults: any;
+}
+
+export class SurveyQuestionReactComponent extends React.Component<Props, State> {
+    constructor(props: Props) {
+        super(props);
+        this.state = {
+            showAnswers: false,
+            questionWithResults: null
+        };
+    }
+
+    showAnswers(e?) {
+        if (e && e.key !== 'Enter') {
+            return;
+        }
+        if (!this.state.questionWithResults.countAnalysis) {
+            this.getAnalysis();
+        } else {
+            this.setState({
+                showAnswers: !this.state.showAnswers
+            });
+        }
+    }
+
+    getAnalysis() {
+        api.getSurveyQuestionResults(this.props.surveyConceptId, this.props.question.conceptId, this.props.question.path)
+            .then(
+                results => {
+                    const questionWithResults = {
+                        countAnalysis: results.items.filter(a => a.analysisId === 3110)[0],
+                        genderAnalysis: results.items.filter(a => a.analysisId === 3111)[0],
+                        ageAnalysis: results.items.filter(a => a.analysisId === 3112)[0],
+                        participantCountAnalysis: results.items.filter(a => a.analysisId === 3203)[0],
+                        versionAnalysis: results.items.filter(a => a.analysisId === 3113)[0]
+                        // this.processResults(q, this.survey.participantCount);
+                    };
+                    questionWithResults.countAnalysis.results.sort((a1, a2) => {
+                        if (a1.countValue > a2.countValue) {
+                            return -1;
+                        }
+                        if (a1.countValue < a2.countValue) {
+                            return 1;
+                        }
+                        return 0;
+                    });
+                    this.setState({ questionWithResults: questionWithResults });
```

Comment:
Need to set `showAnswers` here for it to show the first time in:
```suggestion
                    this.setState({ questionWithResults: questionWithResults, showAnswers: true });
```